<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QDialog" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edytor wpisu</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Wpis:</string>
       </property>
       <property name="buddy">
        <cstring>entryTextField</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="entryTextField">
       <property name="minimumSize">
        <size>
         <width>244</width>
         <height>221</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ustaw date wpisu:</string>
       </property>
       <property name="buddy">
        <cstring>entryDateSelector</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="entryDateSelector"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ustaw format wpisu:</string>
       </property>
       <property name="buddy">
        <cstring>entryFormatSelector</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="entryFormatSelector"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Ustaw kolor wpisu:</string>
       </property>
       <property name="buddy">
        <cstring>entryColorSelector</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="entryColorSelector"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Dodaj zdjęcie do wpisu:</string>
       </property>
       <property name="buddy">
        <cstring>addPhotoButton</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addPhotoButton">
       <property name="text">
        <string>Dodaj zdjęcie</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
         <property name="centerButtons">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>entryTextField</tabstop>
  <tabstop>entryDateSelector</tabstop>
  <tabstop>entryColorSelector</tabstop>
  <tabstop>addPhotoButton</tabstop>
  <tabstop>entryFormatSelector</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Editor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Editor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
